notebook/agenteval_cq_math.ipynb

```diff
@@ -91,67 +158,16 @@
     "\n",
     "import autogen\n",
     "\n",
-    "config_list = autogen.config_list_from_json(\n",
-    "    \"OAI_CONFIG_LIST\",\n",
-    "    filter_dict={\n",
-    "        \"model\": [\"gpt-4\"],\n",
-    "    },\n",
-    ")"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {
-    "id": "fBZ-XFXy1_BJ"
-   },
-   "source": [
-    "\n",
-    "## Construct `CriticAgent`\n",
-    "\n",
-    "We construct the planning agent named `critic` and a user proxy agent for the critic named `critic_user`. We specify `human_input_mode` as \"NEVER\" in the user proxy agent, ensuring that it will never ask for human feedback. Additionally, we define the `ask_critic` function to send a message to the critic and retrieve the criteria from the critic.\n"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 3,
-   "metadata": {
-    "id": "9XAeyjd11_BK"
-   },
-   "outputs": [],
-   "source": [
-    "critic = autogen.AssistantAgent(\n",
-    "    name=\"critic\",\n",
-    "    llm_config={\"config_list\": config_list},\n",
-    "    system_message=\"\"\"You are a helpful assistant. You suggest criteria for evaluating different tasks. They should be dinstinguishable, quantifieable and not redundant.\n",
-    "    Convert the evaluation criteria into a dictionary where the keys are the criteria.\n",
-    "    The value of each key is a dictionary as follows {\"description\": criteria description , \"accepted_values\": possible accepted inputs for this key}\n",
-    "    Make sure the keys are criteria for assessing the given task.  \"accepted_values\" include the acceptable inputs for each key that are fine-grained and preferably multi-graded levels. \"description\" includes the criterion description.\n",
-    "    Return the dictionary.\"\"\",\n",
-    ")\n",
-    "\n",
-    "critic_user = autogen.UserProxyAgent(\n",
-    "    name=\"critic_user\",\n",
-    "    max_consecutive_auto_reply=0,  # terminate without auto-reply\n",
-    "    human_input_mode=\"NEVER\",\n",
-    "    code_execution_config={\n",
-    "        \"use_docker\": False\n",
-    "    },  # Please set use_docker=True if docker is available to run the generated code. Using docker is safer than running the generated code directly.\n",
-    ")\n",
-    "\n",
-    "\n",
-    "def ask_critic(message):\n",
-    "    \"\"\"\n",
-    "    Initiate a chat with the critic user and return the last message received from the planner.\n",
+    "sys.path.insert(0, os.path.abspath(os.path.join(\"..\", \"autogen\", \"agentchat\", \"contrib\")))\n",
```

Comment:
use an import statement from the subpackage, like `from autogen.agentchat.contrib.critic_agent import CriticAgent`